### PR TITLE
Wrote another few pages of the specification and added the use of a t…

### PR DESCRIPTION
**Pull request by kiniry**
Note that this is branched off of `protocol-spec`.  

I'm asking @mkiniry to review and merge to give him practice with such.

### CODE REVIEW

File: specs/pvs/corla.pvs [hunk 1]
```diff
@@ -94,33 +101,156 @@
...
+%%% \section{Kinds of Concepts}
+%%%
+%%% There are basically three kinds of concepts, each of which is a
+%%% part of speech, that are introduced in any specification:
+%%% \begin{itemize}
+
+%%% \item \textbf{Nouns} are formalized by regular (composite)
+%%% types. The set of types available in the prelude of PVS is large,
+%%% ranging from booleans to real-world concepts like time and
+%%% tokenizing leers. Composite types are product types such as tuples
```
Comment (nealmcb) [inline specs/pvs/corla.pvs]: tokenizing _whats_?

File: specs/pvs/corla.pvs [hunk 2]
```diff
@@ -636,9 +767,10 @@ BEGIN
 % Note that https://github.com/usnistgov/ElectionResultsReporting/ defines
 % it as "Undervote: Occurs when the voter does not select a candidate in
 % a 1-of-M contest or selects fewer than N candidates in an N-of-M contest."
-% @todo kiniry Should we model a blank vote separately from an
-% undervote?  Or do they simply use the term blank vote to denote an
-% undervote? @trace CDOS Audit Board Utility Mock-up 21 July 2017
+%%% \todojrk{Should we model a blank vote separately from an
+%%% undervote?  Or do they simply use the term blank vote to denote an
+%%% undervote?}
```
Comment (nealmcb) [inline specs/pvs/corla.pvs]: As explained in a email from Dwight 2017-07-27, they use the term "blank vote" for zero votes, and "undervote" for "less than votes_allowed votes".

File: specs/pvs/corla.pvs [hunk 3]
```diff
@@ -1,55 +1,61 @@
...
-%%% written in PVS's higher-order logic (HOL). Our informal domain
-%%% model is written in the Free \& Fair System Specification Language,
-%%% or FAFESSL for short. FAFESSL is a daughter of Extended BON which
-%%% was, in turn, a daughter of BON, the Business Object Notation.
+%%%
+%%% This document is written in a literate style using the PVS theorem
+%%% prover. Writing system specifications in this fashion is called
+%%% (formal) domain engineering and system specification. Our formal
+%%% domain model is written in PVS's higher-order logic (HOL). Our
+%%% informal domain model is written in the Free \& Fair System
```
Comment (mkiniry) [inline specs/pvs/corla.pvs]: Is the \ before the & there on purpose?

File: specs/pvs/corla.pvs [hunk 4]
```diff
@@ -1,55 +1,61 @@
...
+%%% model specification.
 %%%
 %%% As mentioned above, we write formal specifications using a
 %%% literate style, a la Knuth. Doing so permits us to document our
 %%% thinking and the system design, from domain modeling and
 %%% engineering all of the way to formal specification and
-%%% verification, in a fashion that produces beautifully printed
-%%% books and interactive hypertext.
+%%% verification, in a fashion that produces beautifully printed books
+%%% and interactive hypertext.
```
Comment (mkiniry) [inline specs/pvs/corla.pvs]: What was the purpose of this edit?

File: specs/pvs/corla.pvs (hunk 3 above)
Comment (dmzimmerman) [inline specs/pvs/corla.pvs]: Yes. If it's not there, LaTeX will crash.x

File: specs/pvs/corla.pvs [hunk 5]
```diff
@@ -94,33 +101,156 @@
...
+%%% non-technical artifacts (statue, rules, contract, etc.) is
+%%% captured using a traceability annotation of the form
+%%% \begin{center}
+%%% \begin{alltt}
+%%% @trace <SOURCE ARTIFACT> Discussion of traceability relationship.
+%%% \end{alltt}
+%%% \end{center}
+%%%
+%%% If a concept is introduced and it has no explicitly annotated
+%%% traceability relation, then the reason for its introduce should be
```
Comment (mkiniry) [inline specs/pvs/corla.pvs]: Reason for its _introduction_, or is 'introduce' correct here?

File: specs/pvs/corla.pvs (hunk 3 above)
Comment (mkiniry) [inline specs/pvs/corla.pvs]: Gotcha.

**Comment by mkiniry** [comment]
I have reviewed this and made a few notes.

File: specs/pvs/corla.pvs (hunk 4 above)
Comment (nealmcb) [inline specs/pvs/corla.pvs]: I'm guessing that the paragraph was automatically re-justified, perhaps along with others nearby, or during proposed edits.
I'd prefer that these sorts of changes be avoided, even if lines are left shorter than possible.

File: specs/pvs/corla.pvs (hunk 5 above)
Comment (nealmcb) [inline specs/pvs/corla.pvs]: Looks like a typo for Introduction, as you suggest.

**Pull request merged**